### PR TITLE
Use status-im/go-web3js

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -316,6 +316,12 @@
   packages = ["."]
   revision = "ed27b6fd65218132ee50cd95f38474a3d8a2cd12"
 
+[[projects]]
+  name = "github.com/status-im/go-web3js"
```
Comment (mandrigin) [inline Gopkg.lock:324]: Can you add a short description of this dependency right into this file?
Maybe with even a link to README (looks like overkill, but it makes it more convenient when one needs to regenerate and update dependency).

**Review by mandrigin — CHANGES_REQUESTED** [review]
Looks good, a tiny little comment would be nice to have.

**Comment by dshulyak** [comment]
@mandrigin added a comment to Gopkg.toml

**Review by mandrigin — APPROVED** [review]

**Comment by mandrigin** [comment]
Looks good now!

**Review by adriacidre — APPROVED** [review]

**Review by themue — APPROVED** [review]
Nice one. 👌

**Comment by dshulyak** [comment]
anyone noticed if this change passed ci? :)

**Comment by mandrigin** [comment]
@dshulyak nope, but what is the point of the CI if it doesn't block merging?..

**Pull request merged**